### PR TITLE
Add MechComp integration to the TEG blowers

### PR DESCRIPTION
**Pull request by TealSeer**
[FEATURE] [STATION SYSTEMS]
## About the PR 

Adds a "toggle active" and "set target pressure" input to the TEG blowers. Switches opening the blower UI from click with multitool to click with empty hand because of the overlap with the mechcomp connection popup.

## Why's this needed? 

Opens the door for more automation setups with the TEG in the same vein as the nuclear reactor, which I think is a cool direction for engineering and gives more reason to integrate MechComp into stuff.

## Changelog 

```changelog
(u) TealSeer
(*) The blowers on the TEG can now be controlled by MechComp. Accessing the blower UI has been changed to clicking with an empty hand.
```

### CODE REVIEW

**Comment by TealSeer** [comment]
> "empty hand" - were silicons addressed in this?

Functions as expected, silicons can access menu with a click, including from a range

File: code/modules/power/TEG.dm [hunk 1]
```diff
@@ -99,6 +103,17 @@
...
 
+	proc/toggle_active()
+		src.target_pressure_enabled = !src.target_pressure_enabled
+		logTheThing(LOG_STATION, src, "toggled blower power [src.target_pressure_enabled ? "on" : "off"] using mechcomp.")
+
+	proc/set_target_pressure(datum/mechanicsMessage/input)
+		if(!length(input.signal)) return
+		var/newpressure = text2num(input.signal)
+		if(!isnum_safe(newpressure) || newpressure == src.target_pressure) return
+		src.target_pressure = clamp(newpressure, 0, 1e5)
```
Comment (TobleroneSwordfish) [inline code/modules/power/TEG.dm:114]: this 1e5 feels like it should be a var or a define?

**Review by TobleroneSwordfish — APPROVED** [review]

File: code/modules/power/TEG.dm [hunk 2]
```diff
@@ -99,6 +103,17 @@
 			src.serial_num += "-[variant_b]"
 			variant_b_active = TRUE
 
+	proc/toggle_active()
+		src.target_pressure_enabled = !src.target_pressure_enabled
+		logTheThing(LOG_STATION, src, "toggled blower power [src.target_pressure_enabled ? "on" : "off"] using mechcomp.")
```
Comment (TheTyrant25) [inline code/modules/power/TEG.dm:108]: Maybe the proc name should be more specific if it's exclusively for mechcomp? Like `mechcomp_toggle_active()` or something

Just future-proofing so no one accidentally uses it for something else (which is extraordinarily unlikely but still)

**Review by TheTyrant25 — APPROVED** [review]
lgtm

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**